2022-03-29 09:42:44,451 INFO    MainThread:293099 [wandb_setup.py:_flush():69] setting env: {}
2022-03-29 09:42:44,451 INFO    MainThread:293099 [wandb_setup.py:_flush():69] setting login settings: {}
2022-03-29 09:42:44,451 INFO    MainThread:293099 [wandb_init.py:_log_setup():338] Logging user logs to /lila/home/harrigan/damuta/docs/examples/wandb/run-20220329_094244-1fvca2u6/logs/debug.log
2022-03-29 09:42:44,452 INFO    MainThread:293099 [wandb_init.py:_log_setup():339] Logging internal logs to /lila/home/harrigan/damuta/docs/examples/wandb/run-20220329_094244-1fvca2u6/logs/debug-internal.log
2022-03-29 09:42:44,453 INFO    MainThread:293099 [wandb_init.py:_jupyter_setup():289] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fcaab57cc10>
2022-03-29 09:42:44,454 INFO    MainThread:293099 [wandb_init.py:init():371] calling init triggers
2022-03-29 09:42:44,454 INFO    MainThread:293099 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-03-29 09:42:44,454 INFO    MainThread:293099 [wandb_init.py:init():420] starting backend
2022-03-29 09:42:44,455 INFO    MainThread:293099 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-29 09:42:44,481 INFO    MainThread:293099 [backend.py:ensure_launched():135] starting backend process...
2022-03-29 09:42:44,496 INFO    MainThread:293099 [backend.py:ensure_launched():139] started backend process with pid: 294474
2022-03-29 09:42:44,499 INFO    MainThread:293099 [wandb_init.py:init():425] backend started and connected
2022-03-29 09:42:44,510 INFO    MainThread:293099 [wandb_run.py:_label_probe_notebook():846] probe notebook
2022-03-29 09:42:44,511 INFO    MainThread:293099 [wandb_run.py:_label_probe_notebook():853] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-29 09:42:44,511 INFO    MainThread:293099 [wandb_init.py:init():473] updated telemetry
2022-03-29 09:42:44,512 INFO    MainThread:293099 [wandb_init.py:init():492] communicating current version
2022-03-29 09:42:45,340 INFO    MainThread:293099 [wandb_init.py:init():497] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-29 09:42:45,342 INFO    MainThread:293099 [wandb_init.py:init():505] communicating run to backend with 30 second timeout
2022-03-29 09:42:45,630 INFO    MainThread:293099 [wandb_init.py:init():530] starting run threads in backend
2022-03-29 09:42:47,439 INFO    MainThread:293099 [wandb_run.py:_console_start():1625] atexit reg
2022-03-29 09:42:47,441 INFO    MainThread:293099 [wandb_run.py:_redirect():1499] redirect: SettingsConsole.WRAP
2022-03-29 09:42:47,442 INFO    MainThread:293099 [wandb_run.py:_redirect():1536] Wrapping output streams.
2022-03-29 09:42:47,850 INFO    MainThread:293099 [wandb_run.py:_redirect():1560] Redirects installed.
2022-03-29 09:42:47,850 INFO    MainThread:293099 [wandb_init.py:init():555] run started, returning control to user process
2022-03-29 09:42:47,855 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:42:47,855 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:42:48,337 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:42:48,831 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:42:48,832 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:42:59,856 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:42:59,867 INFO    MainThread:293099 [wandb_run.py:_config_callback():874] config_cb None None {'opt_method': 'ADVI', 'seed': 2021, 'n_sigs': 20, 'alpha_bias': 0.1, 'psi_bias': 0.01, 'n': 100, 'init_strategy': 'kmeans'}
2022-03-29 09:43:40,629 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:43:40,633 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:44:02,309 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:44:02,322 INFO    MainThread:293099 [wandb_run.py:_config_callback():874] config_cb None None {'n_damage_sigs': 8, 'n_misrepair_sigs': 5, 'beta_bias': 0.1, 'gamma_bias': 0.01}
2022-03-29 09:45:13,838 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:45:13,838 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:45:14,120 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:45:14,134 INFO    MainThread:293099 [wandb_run.py:_config_callback():874] config_cb None None {}
2022-03-29 09:46:14,242 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:46:14,243 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:46:40,807 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:46:40,814 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:46:40,815 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:46:58,534 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:46:58,540 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:46:58,540 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:47:07,671 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:47:07,679 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:47:07,679 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:47:18,447 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:47:18,458 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:47:18,459 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:48:38,927 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:48:38,939 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:48:38,939 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:49:18,649 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:49:18,695 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:49:18,695 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:49:29,847 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:49:29,857 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:49:29,857 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:50:03,290 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:50:03,420 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:50:03,420 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:51:31,244 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:51:31,255 INFO    MainThread:293099 [wandb_run.py:_config_callback():874] config_cb None None {}
2022-03-29 09:52:34,223 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:52:34,224 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:52:37,458 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:52:37,498 INFO    MainThread:293099 [wandb_run.py:_config_callback():874] config_cb None None {}
2022-03-29 09:52:37,525 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:52:37,525 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:52:40,248 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:52:40,267 INFO    MainThread:293099 [wandb_run.py:_config_callback():874] config_cb None None {}
2022-03-29 09:52:40,286 INFO    MainThread:293099 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:52:40,286 INFO    MainThread:293099 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:52:46,535 INFO    MainThread:293099 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:52:46,558 INFO    MainThread:293099 [wandb_setup.py:_flush():69] setting env: {}
2022-03-29 09:52:46,558 INFO    MainThread:293099 [wandb_setup.py:_flush():69] setting login settings: {}
2022-03-29 09:52:46,558 INFO    MainThread:293099 [wandb_init.py:_log_setup():338] Logging user logs to /lila/home/harrigan/damuta/docs/examples/wandb/run-20220329_095246-lmo1f6yi/logs/debug.log
2022-03-29 09:52:46,559 INFO    MainThread:293099 [wandb_init.py:_log_setup():339] Logging internal logs to /lila/home/harrigan/damuta/docs/examples/wandb/run-20220329_095246-lmo1f6yi/logs/debug-internal.log
2022-03-29 09:52:46,559 INFO    MainThread:293099 [wandb_init.py:init():371] calling init triggers
2022-03-29 09:52:46,559 INFO    MainThread:293099 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-03-29 09:52:46,559 INFO    MainThread:293099 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 1fvca2u6
2022-03-29 09:52:46,561 INFO    MainThread:293099 [wandb_run.py:finish():1295] finishing run harrig12/damuta-docs_examples/1fvca2u6
